app/handlers/sessions_create.rb

```diff
@@ -224,12 +212,8 @@ def authentication_user
     @authentication_user ||= authentication.user
   end
 
-  def signing_up?
-    pre_auth_state.present?
-  end
-
-  def pre_auth_state
-    options[:pre_auth_state]
+  def signing_in?
+    buybug
```

Comment:
buybug?

Reply:
:-)

`byebug` 

But of course, we don't want it in here anyway :-)